md/src/arithmetic/CozaLozaWoza.java

```diff
@@ -0,0 +1,19 @@
+package io.codelex.arithmetic;
+
+public class CozaLozaWoza {
+    public static void main(String[] args) {
+        for(int i = 0; i <= 109; i++) {
+            if (i % 11 == 0) {
+                System.out.print("\n");
+            }
+            String x = Integer.toString(i + 1);
+            if ((i + 1) % 3 == 0) {
```

Comment:
Drīzāk ciklam vajdzētu iet no `int i = 1` un līdz 110, jo tad nebūs visur jāraksta `i+1`, kā arī neizskatās, ka šis strādātu līdz galam pareizi - jo ja dalās, piemēram, gan ar 3, gan ar 5 tad cipara cietā tiks izvadīts tikai Coza nevis CozaLoza.